FEM/rf_tim_new.h

```diff
@@ -143,6 +148,15 @@ class CTimeDiscretization
     double GetATol() const { return absolute_error; }
     double GetMaximumTSizeRestric() const { return h_max; }
     double GetMinimumTSizeRestric() const { return h_min; }
+
+    double GetRejectedFator() const { return _reject_factor; }
+    void SetOldStepSize(const double step_size)
+        { _old_step_size = step_size; }
+    double GetMinStepSize() const { return _min_step_size; }
+    double GetMaxStepSize() const { return _max_step_size; }
+    double limitStepSizeByIncrementRatio(const double  h) const;
```

Comment:
is it possible to start the function name with a capital letter? (like other function names above)

Reply:
changed.